backend/src/main/kotlin/code/nebula/cipherquest/configuration/ChatClientConfiguration.kt

```diff
@@ -55,30 +46,22 @@ class ChatClientConfiguration(
                 SanitizeInputAdvisor(),
                 ProtectedInputAdvisor(vectorStore, vectorStoreService),
                 LevelUpAdvisor(vectorStore, userLevelService, messageContext),
-                VectorStoreChatMemoryAdvisor(vectorStore, memorySystemText, chatHistoryWindowSize),
+                VectorStoreChatMemoryAdvisor
+                    .builder(vectorStore)
+                    .chatMemoryRetrieveSize(gameConfig.ai.chat.historyMaxSize)
+                    .build(),
```

Comment:
We are not sending anymore the chat memory system text.

We wanna use the default one?

Reply:
We already using the default one